README.md

```diff
@@ -52,7 +52,7 @@ zlib1g-dev
 Inside the `repository root` use the following commands:
 
 ```sh
-./patchGenieutils.sh
+patch -d genieutils < genieutils.patch
 mkdir build
 cd build
 cmake ..
```

Comment:
```suggestion
cmake -DSTATIC_COMPILE=TRUE ..
```

Reply:
This still needs to be added as a flag for CMake

Reply:
And `genieutils` submodule needs to be updated to commit `6297d093db063bb0c6a3d69a3cc82aebfa88a1c6`